assets/stylesheets/components/_notification-banner.scss

```diff
@@ -15,6 +15,32 @@
   // scss-lint:enable SelectorFormat
 }
 
+.notification-banner__hiviz {
+  background-color: tint($yellow, 25%);
+
+  a {
```

Comment:
I think this is a fair compromise and am happy to adjust things like link colours based on the background colour.

Only thing I'm not sure on is the `tint` and `shade` mixins now. We might want to think about the colour options and what shades we want to use and keeping the spacing consistent rather than just allowing things like 18% here and 22% lighter somewhere else?